metadata-ingestion/src/datahub/ingestion/source/airbyte/rest_api_wrapper/data_resolver.py

```diff
@@ -0,0 +1,434 @@
+
+    def _get_connections(self, workspace_id: str) -> List[Connection]:
+        """
+        Get the list of connections from Airbyte Cloud
+        """
+        connections_list_endpoint: str = self._get_connections_endpoint(workspace_id)
+        logger.debug(f"Request to URL={connections_list_endpoint}")
+        response = self._request_session.get(
+            connections_list_endpoint,
+            headers=self._get_authorization_header(),
+        )
+        response.raise_for_status()
+        connections = response.json().get(Constant.DATA)
+        return self._get_connections_from_response(connections) if connections else []
+
+    def _get_source(self, source_id: str) -> Source:
+        """
+        Get the source details from Airbyte Cloud
+        """
+        source_endpoint: str = self._get_source_endpoint(source_id)
+        logger.debug(f"Request to URL={source_endpoint}")
+        response = self._request_session.get(
+            source_endpoint,
+            headers=self._get_authorization_header(),
+        )
+        response.raise_for_status()
+        return self._get_source_from_response(response.json())
+
+    def _get_destination(self, destination_id: str) -> Destination:
+        """
+        Get the destination details from Airbyte Cloud
+        """
+        destination_endpoint: str = self._get_destination_endpoint(destination_id)
+        logger.debug(f"Request to URL={destination_endpoint}")
+        response = self._request_session.get(
+            destination_endpoint,
+            headers=self._get_authorization_header(),
+        )
+        response.raise_for_status()
+        return self._get_destination_from_response(response.json())
+
+    def _get_jobs(self, connection_id: str) -> List[Job]:
+        """
+        Get the jobs list from Airbyte Cloud
+        """
+        jobs_endpoint: str = self._get_jobs_endpoint(connection_id)
+        logger.debug(f"Request to URL={jobs_endpoint}")
+        response = self._request_session.get(
+            jobs_endpoint,
+            headers=self._get_authorization_header(),
+        )
+        response.raise_for_status()
+        jobs = response.json().get(Constant.DATA)
+        if jobs:
+            for job in jobs:
+                # Convert string to timestamp
+                start_time = datetime.strptime(
+                    job[Constant.STARTTIME], "%Y-%m-%dT%H:%M:%SZ"
+                )
+                job[Constant.STARTTIME] = round(start_time.timestamp())
+                # Convert ISO_8601 format duration to timedelta
+                # And add it with start time to get ent time. Ex: 'PT1M10S' to
+                end_time = start_time + parse_duration(job[Constant.DURATION])
+                job[Constant.ENDTIME] = round(end_time.timestamp())
+            return self._get_jobs_from_response(jobs)
+        else:
+            return []
+
+
+class OssAPIResolver(DataResolverBase):
+    BASE_URL = "http://localhost:8000/api/v1"
```

Comment:
take this from config and keep default to this value

Reply:
Done